proto/decentraland/common/texture.proto

```diff
@@ -20,10 +20,8 @@ message Texture {
   optional TextureFilterMode filter_mode = 3; // default = FilterMode.Bilinear
 
   // Final uv = offset + (input_uv * tiling)
-  // Offset for texture positioning. 
-  optional Vector2 offset = 4; // default = Vector2.Zero
-  // Tiling multiplier for texture repetition.
-  optional Vector2 tiling = 5; // default = Vector2.One
+  optional Vector2 offset = 4; // default = Vector2.Zero; Offset for texture positioning, only works for the texture property in PbrMaterial or UnlitMaterial.
+  optional Vector2 tiling = 5; // default = Vector2.One; Tiling multiplier for texture repetition, only works for the texture property in PbrMaterial or UnlitMaterial.
```

Comment:
I'm not totally sure about keeping the extra info about other components here. For now, it's better to keep it than not having  it at all

Reply:
The idea was to have that documentation for creators when working with `alpha_texture`, `emissive_texture`, `bump_texture`, since the tiling and offset properties would show up with the autocomplete.